src/pages/Library/index.js

```diff
@@ -1,79 +1,112 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { collection, getDocs } from 'firebase/firestore'
+import { db } from '../../utils/firestore'
+import { useUserAuth } from "../../context/userAuthContext"
+import 'holderjs'
+import Button from 'react-bootstrap/Button'
+import Carousel from 'react-bootstrap/Carousel'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
 
-import 'holderjs';
+import Book from '../../components/Book'
 
-import Button from 'react-bootstrap/Button';
-import Card from 'react-bootstrap/Card';
+import './style.css'
 
-import Carousel from 'react-bootstrap/Carousel';
+const Library = () => {
 
-import './style.css'
+  const [books, setBooks] = useState() //[{items:[]}])
+
+  const { user } = useUserAuth()
+  const { user: { uid } } = user;
+
+  const fetchBooks = async () => {
+    console.log('[Library] fetchBooks')
+    const collectionRef = collection(db, 'library', uid, 'books')
+
+    const booksSnapshot = await getDocs(collectionRef)
+    let items = [];
+    if (booksSnapshot.docs.length > 0) {
+      booksSnapshot.forEach((doc) => {
+        console.log('[Library] booksRef - doc ', doc.id, doc.data())
+        items.push(doc.data().item)
+      })
+    }
+
+    // const querySnapshot = await getDocs(collectionRef)
+    // const books = querySnapshot.docs.map((d) => ({ id: d.id, ...d.data() }))
```

Comment:
same, can this be removed if not required

Reply:
Done